Shell/LineEditor.cpp

```diff
@@ -226,6 +226,23 @@ String LineEditor::get_line(const String& prompt)
                 fflush(stdout);
                 continue;
             }
+            if (ch == 0x01) // ^A
+            {
+                if (m_cursor > 0) {
+                    fprintf(stdout, "\033[%dD", m_cursor);
+                    fflush(stdout);
+                    m_cursor = 0;
+                }
+                continue;
+            } else if (ch == 0x05) // ^E
```

Comment:
No need for `else` after a `continue`.

```suggestion
            }
            if (ch == 0x05) { // ^E
```
```